metadata-ingestion/src/datahub/ingestion/source/aws/glue.py

```diff
@@ -1189,5 +1291,56 @@ def get_data_platform_instance() -> DataPlatformInstanceClass:
     def get_report(self):
         return self.report
 
+    def create_checkpoint(self, job_id: JobId) -> Optional[Checkpoint]:
+        """
+        Create the custom checkpoint with empty state for the job.
+        """
+        assert self.ctx.pipeline_name is not None
+        if job_id == self.get_default_ingestion_job_id():
+            return Checkpoint(
+                job_name=job_id,
+                pipeline_name=self.ctx.pipeline_name,
+                platform_instance_id=self.get_platform_instance_id(),
```

Comment:
It is not clear to me why a platform instance is mandatory for stateful ingestion for glue